docs/tutorial/get-started.md

```diff
@@ -55,14 +55,13 @@ If you want to learn about Scylla, please refer to [the official website](https:
     Deploy Scylla as a backup database.
 
     ```bash
-    kubectl apply -f k8s/jobs/db/initialize/cassandra/configmap.yaml
-    kubectl apply -f k8s/external/scylla
+    make k8s/external/scylla/deploy
     ```
 
     Apply kubernetes metrics-server
 
     ```bash
-    kubectl apply -f k8s/metrics/metrics-server
+    make k8s/metrics/metrics-server/deploy
```

Comment:
metrics-server deploy command is simple, as you [mentioned](https://github.com/vdaas/vald/pull/909/files/82fd3963055b909cdf833b65317d4c09603cba99?file-filters%5B%5D=.json&file-filters%5B%5D=.md&file-filters%5B%5D=.mk&file-filters%5B%5D=.tpl&file-filters%5B%5D=.yaml#r553701026) we can write kubectl apply.